### ISSUE TITLE
eyeglass 0.8 throws a warning with sass-mq

### ISSUE THREAD

**Issue by PJL101** [sass-mq/sass-mq#79]
Hi,

Every time you run node-sass, eyeglass produces an error saying:

```
The following modules did not declare an eyeglass version:
  sass-mq
Please add the following to the module's package.json:
  "eyeglass": { "needs": "^0.8.2" }
```

After looking at the eyeglass documentation, I have created a PR https://github.com/sass-mq/sass-mq/pull/78 which adds required information to the package.json. I have also bumped the required eyeglass version because of this change.

I believe this is correct but let me know if I need to make any changes.

**Comment by kaelig** [comment]
Hi, thank you so much for submitting this fix.

I already had a branch (see `update-dependencies` in #77)  but did not do a release yet.

Can you try installing sass-mq with `npm install sass-mq/sass-mq#update-dependencies --save-dev` and let me know if it works for you?

**Comment by PJL101** [comment]
Apologies @kaelig I didn't see the branch.

Tested the above locally and on Travis - https://travis-ci.org/PJL101/foley/builds/105909333, confirmed fixed.

**Comment by kaelig** [comment]
Thank you, I'll make a release then!

**Comment by kaelig** [comment]
Fixed in 3.2.6.

**Comment by PJL101** [comment]
Great, thanks for looking at this so quickly!
